demisto_sdk/commands/convert/README.md

```diff
@@ -0,0 +1,30 @@
+## convert
+
+**Convert layouts and classifiers from a new version to an old version and vice versa.**
+
+##Note:
+**Classifiers/Mappers conversion from 6.0.0 and above to versions 5.9.9 and below is currently not supported. Request for converting a pack for versions below 6.0.0 will result on layouts conversion only.**
```

Comment:
We should really create a simple list of what can be converted, something that is easy to read and gets the info out quickly the first time you approach the command.